flow/project.py

```diff
@@ -1764,7 +1818,7 @@ def print_status(self, jobs=None, overview=True, overview_max_lines=None,
             ]
 
             with prof(single=False):
-                tmp = self._fetch_status(jobs, err, ignore_errors, no_parallelize)
```

Comment:
We still have the argument `no_paralleiize` in `print_status`'s function signature. We should issue a `DeprecationWarning` if it is set to True and change the  configuration value for the parallelization to `none`. We also need to change the docstring to note that the argument is official deprecated.

Reply:
I'll edit the docstring. And will deprecate `no_parallelize` and edit the code